src/js/components/icons/status/ErrorStatus.js

```diff
@@ -2,19 +2,31 @@
 
 var React = require('react');
 var IntlMixin = require('../../../mixins/GrommetIntlMixin');
+var uuid = require('node-uuid');
 
 var ErrorStatus = React.createClass({
 
   mixins: [IntlMixin],
 
+  propTypes: {
+    a11yTitle: React.PropTypes.string
+  },
+
   render: function() {
     var className = 'status-icon status-icon-error';
+    var a11yTitle;
     if (this.props.className) {
       className += ' ' + this.props.className;
     }
+    if (typeof this.props.a11yTitle === "undefined") {
```

Comment:
Same here:

`a11yTitle = this.props.a11yTitle || this.getGrommetIntlMessage('Error')`